rosa_architecture/rosa_policy_service_definition/rosa-hcp-instance-types.adoc

```diff
@@ -9,7 +9,7 @@ toc::[]
 
 [NOTE]
 ====
-Currently, {hcp-title} supports a maximum of 250 worker nodes.
+Currently, {hcp-title} supports a maximum of 250 worker nodes. Customers with {hcp-title} 4.14.x and 4.15.x clusters require a minimum z-stream version of 4.14.28 or 4.15.15 and greater to scale to 250 worker nodes. For earlier versions, the maximum is 90 worker nodes. 
```

Comment:
```suggestion
Currently, {hcp-title} supports a maximum of 250 worker nodes. Customers using  {hcp-title} 4.14.x and 4.15.x clusters require a minimum z-stream version of 4.14.28 or 4.15.15 and greater to scale to 250 worker nodes. For earlier versions, the maximum is 90 worker nodes. 
```

Reply:
I suggest this merely to avoid repeating the word "with" two times right together.

Reply:
The same is true of the other NOTES, two instances of the word "with" close together, because {hcp-title} renders to "ROSA with HCP".